[2024-05-09T02:06:15.473+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-09T02:06:15.511+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_v1.training_model_A manual__2024-05-09T02:06:11.499284+00:00 [queued]>
[2024-05-09T02:06:15.531+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_v1.training_model_A manual__2024-05-09T02:06:11.499284+00:00 [queued]>
[2024-05-09T02:06:15.532+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-09T02:06:15.558+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): training_model_A> on 2024-05-09 02:06:11.499284+00:00
[2024-05-09T02:06:15.575+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3378) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-09T02:06:15.592+0000] {standard_task_runner.py:63} INFO - Started process 3384 to run task
[2024-05-09T02:06:15.598+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_dag_v1', 'training_model_A', 'manual__2024-05-09T02:06:11.499284+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpz1drsgut']
[2024-05-09T02:06:15.608+0000] {standard_task_runner.py:91} INFO - Job 33: Subtask training_model_A
[2024-05-09T02:06:15.724+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag_v1.training_model_A manual__2024-05-09T02:06:11.499284+00:00 [running]> on host 2b3626ba2b72
[2024-05-09T02:06:15.920+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='satyendra' AIRFLOW_CTX_DAG_ID='my_dag_v1' AIRFLOW_CTX_TASK_ID='training_model_A' AIRFLOW_CTX_EXECUTION_DATE='2024-05-09T02:06:11.499284+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-09T02:06:11.499284+00:00'
[2024-05-09T02:06:15.924+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-09T02:06:15.926+0000] {python.py:237} INFO - Done. Returned value was: 2
[2024-05-09T02:06:15.927+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-09T02:06:15.966+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=my_dag_v1, task_id=training_model_A, execution_date=20240509T020611, start_date=20240509T020615, end_date=20240509T020615
[2024-05-09T02:06:16.009+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-09T02:06:16.042+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-09T02:06:16.045+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
